packages/gis/lua/Layer.lua

```diff
@@ -483,6 +483,16 @@ Layer_ = {
 	delete = function(self)
 		deleteData(self)
 	end,
+	--- Drop a database.
```

Comment:
Database or layer?

"This function only works when the layer is stored in a PostGIS table."

Reply:
This function works like 'delete' function. It doesn't drop the layer, it just remove the database.

Reply:
"Note that it removes all tables within the database."